CHANGELOG.md

```diff
@@ -7,11 +7,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## 2.0.0 - Jan 30, 2020
+
+This release adds support for the new rippled protocol buffers and adds browser compatibility support. 
+
+### New Protocol Buffers from rippled
+
+rippled implemented protocol buffer support natively, which uses a new and incompatible set of protocol buffers. The implementation in rippled was completed in: ripple/rippled#3159. Xpring-JS is now capable of using these protocol buffers.  This change is opt-in and non-breaking. 
```

Comment:
two spaces after last sentence but no where else in the changelog do we do two spaces.